### PR TITLE
typos

### PR DESCRIPTION
**Pull request by beadle2**
I found a few typos in 2OD.md

### CODE REVIEW

File: docs/Training-Manual/02-Stage-1/2.OD.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ We are getting ready to type text into an existing project. Before you can do th
 The translator who organises his/her desktop well has all the resources necessary for his/her work.
 
 **What are you going to do?**  
-You will start the Paratext 9 program and open a previously saved layout (text combination). If needed you will open other resources, change the arrangement of the windows and resave the text layout.
+You will start the Paratext 9 program and open a previously saved layout (text combination). If needed you will open other Fresources, change the arrangement of the windows and resave the text layout.
```
Comment (beadle2) [inline docs/Training-Manual/02-Stage-1/2.OD.md]: reject

File: docs/Training-Manual/02-Stage-1/2.OD.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ We are getting ready to type text into an existing project. Before you can do th
...
 
 :::info Videos 
-There are a number of videos available to help you with the different types of resources and arranging the windows. Some suggestion are given below. Click on the link to see the video summary.  
+There are a number of videos available to help you with the different types of resources and arranging the windows. Some suggestions are given below. Click on the link to see the video summary.
```
Comment (beadle2) [inline docs/Training-Manual/02-Stage-1/2.OD.md:18]: accept

File: docs/Training-Manual/02-Stage-1/2.OD.md (hunk 1 above)
Comment (JenniBeadle) [reply docs/Training-Manual/02-Stage-1/2.OD.md]: remove f = resources

File: docs/Training-Manual/02-Stage-1/2.OD.md (hunk 2 above)
Comment (JenniBeadle) [reply docs/Training-Manual/02-Stage-1/2.OD.md:18]: accept

**Comment by JenniBeadle** [comment]
accept all

**Pull request merged**